lua/nvim-treesitter/configs.lua

```diff
@@ -139,9 +139,9 @@ parserlist.typescript = {
   }
 }
 
-parserlist.tsx = {
+parserlist.typescriptreact = {
   install_info = {
```

Comment:
Needs to be changed also in base language map.

Reply:
indeed, this is a very much WIP PR until i can make some changes into neovim's core !

Reply:
actually found that neovims core might just work quite fine. I will fix the issues in this PR instead